molecule/commands.py

```diff
@@ -154,15 +155,16 @@ def converge(self, idempotent=False, create_instances=True, create_inventory=Tru
             kwargs.pop('_err', None)
             kwargs['_env']['ANSIBLE_NOCOLOR'] = 'true'
             kwargs['_env']['ANSIBLE_FORCE_COLOR'] = 'false'
-            try:
-                output = sh.ansible_playbook(playbook, *args, **kwargs)
-                return output
-            except sh.ErrorReturnCode as e:
-                print('ERROR: {}'.format(e))
-                sys.exit(e.exit_code)
+
         try:
             output = sh.ansible_playbook(playbook, *args, **kwargs)
-            return output.exit_code
+            if self.molecule._args['--debug']:
```

Comment:
Can we report the output prior to execution.  Would be nice to see what we will be doing vs waiting to converge to display this.